test/DynamoCoreWpfTests/WorkspaceSaving.cs

```diff
@@ -1266,6 +1268,58 @@ public void CustomNodeWorkspaceSavedToSamePlaceNotCausingDuplicatedLibraryItems(
                             x => string.CompareOrdinal(x.Name, nodeName) == 0).Count();
             Assert.AreEqual(count, 2);
         }
+
+        [Test]
+        public void CustomNodeBelongingToPackageCanBeModifiedAndReopenedWithoutError()
+        {
+
+            // load package A
+            // place custom node instance belonging to package A in homeworkspace
+            // modify custom node
+            // save custom node to same path
+            // close custom node
+            // open custom node
+            // assert instances changed their number of output ports.
+            var packageDirectory = Path.Combine(TestDirectory, "pkgs", "PackageThatWillBeModified");
```

Comment:
Does not adding this new package change the number of total packages in the test folder. Unfortunately all tests testing for the total number of packages in this folder may need updating.